src/puppetlabs/puppetdb/pql/transform.clj

```diff
@@ -2,15 +2,21 @@
   (:require [clojure.string :as str]
             [puppetlabs.puppetdb.cheshire :as json]))
 
+(defn paging-clause?
```

Comment:
yeah, instaparse doesn't appear to support that. What I'm reacting to here is that we're building some logic to rebuild something that's already present in the structure of the grammar. Another way to handle this would be to change `<paging-clause>` to `paging-clause`, then have our transformer look for `[:paging-clause ...]` structures. But that may may be sufficiently different from the existing code that we don't want to go there.